lib/bolt_server/transport_app.rb

```diff
@@ -62,11 +62,7 @@ def run_task(target, body)
 
       task = Bolt::Task::PuppetServer.new(body['task'], @file_cache)
       parameters = body['parameters'] || {}
-      results = @executor.run_task(target, task, parameters)
-
-      # Since this will only be on one node we can just return the first result
-      result = scrub_stack_trace(results.first.status_hash)
-      [200, result.to_json]
+      @executor.run_task(target, task, parameters)
```

Comment:
Moving the HTTP response details to the caller here seemed like a simple way to make `run_task` more focused on just the task operations, but open to changing this